### ISSUE TITLE
CanZE IOS version connection problems

### ISSUE THREAD

**Issue by nunosoares73** [fesch/CanZE#608]
Good afternoon,

I bought KONNWEI KW903 Wireless WIFI ELM327 OBD-II Mini Car Auto Diagnostic Scan Tool, but I am not able to connect to the CanZE APP for Iphone.

Some screen shots of my phone and the product:
![IMG_5350](https://user-images.githubusercontent.com/63924814/79650201-f00b9080-81a8-11ea-8267-f5f286f29208.PNG)
![IMG_5349](https://user-images.githubusercontent.com/63924814/79650625-0e718c00-81a9-11ea-846b-b8f54f623cf8.PNG)
![IMG_5348](https://user-images.githubusercontent.com/63924814/79650704-14676d00-81a9-11ea-970b-4cb8ec02e482.PNG)
![IMG_5346](https://user-images.githubusercontent.com/63924814/79650936-26491000-81a9-11ea-8cd7-b9e03efe2446.PNG)
![IMG_5344](https://user-images.githubusercontent.com/63924814/79651167-36f98600-81a9-11ea-9297-78d53b8b6652.PNG)
![IMG_5347](https://user-images.githubusercontent.com/63924814/79653268-d4ed5080-81a9-11ea-95ab-f3a0e44b9eff.jpg)
Can you understand why?

Tks

**Comment by FredLeudon** [comment]
Hi, 

Up address is the dongles IP address.

Like this : 



On settings app, 

Configurar IP : manual
Endereço IP : 192.168.0.11
Máscara ... : 255.255.255.0
Router : leave blank

So when connected to the dongle, your phone will use cellular to go to the web. 

Regards. 


Frédéric RICHARD

**Comment by nunosoares73** [comment]
Hi Frédéric,

First of all thanks for the help.

If I understood what you said, in the CanZe app remove the IP Address from the "router" field and leave it blank. The problem that in the app settings, I can only change the ip address or the port!
The "router" field is not visible.

![IMG_5349](https://user-images.githubusercontent.com/63924814/79690510-b68f5f80-8252-11ea-80a4-01e11c4fdc93.jpg)
In the Iphone settings, I changed the setting to manuel and it looked like this:

![IMG_5352](https://user-images.githubusercontent.com/63924814/79690317-ba6eb200-8251-11ea-844f-4200b1df12a6.PNG)

But the problem remains.

![IMG_5348](https://user-images.githubusercontent.com/63924814/79690358-f1dd5e80-8251-11ea-9a2d-fa6f17b79d45.PNG)

In other words, the CanZe app always assumes the "router" field with the ip address 192.168.0.10

What am I doing wrong?

**Comment by FredLeudon** [comment]
Hi,

Enter 192.168.0.10 in CanZE app. Port would be 35000.

Assume your car is running (key in, break & start pressed)

If CanZE can’t connect, find the test elm button and send log.

Otherwise, there is a v2 still in development.

You can install this public beta here : https://testflight.apple.com/join/nhMX6f5n

Thanks,

Regards.

Frédéric RICHARD

**Comment by nunosoares73** [comment]
Hi Frédéric,

With the normal version of CanZe and following steps:
- Enter 192.168.0.10 in CanZE app. Port would be 35000.
- Car is running (key in, break & start pressed)

The result was:
![IMG_5358](https://user-images.githubusercontent.com/63924814/79697280-bce50200-8279-11ea-9a39-dc9d76c2b91c.PNG)
![IMG_5357](https://user-images.githubusercontent.com/63924814/79697283-c5d5d380-8279-11ea-8e21-f61f052a7ccc.PNG)
![IMG_5356](https://user-images.githubusercontent.com/63924814/79697292-d0906880-8279-11ea-954b-cd8fbdb9a7f8.PNG)

With the beta version:

![IMG_5361](https://user-images.githubusercontent.com/63924814/79697322-fddd1680-8279-11ea-8bf8-22de2d86333b.PNG)
![IMG_5359](https://user-images.githubusercontent.com/63924814/79697334-11887d00-827a-11ea-9711-7b204435c0fc.PNG)
![IMG_5362](https://user-images.githubusercontent.com/63924814/79697360-40065800-827a-11ea-9800-e852a1cd1333.PNG)

While reading the car was on, the foot brake and accelerator on the bottom.

With the beta version there are no more communication problems with the OBD device, however it seems that the application cannot communicate with the car.
I also notice that in the beta version settings there is no field to choose the Zoe model.

Any more suggestions or configuration changes?

Once again, thanks for your help.

Nuno

**Comment by FredLeudon** [comment]
Ok, same issue with the beta version. 

In the beta version, tap the top left icon (sliding menu) and select activities.

Then swipe to « technical data » and « test elm »

It seems your dongle doesn’t respond correctly.



Then go to settings and swipe to find « manage log files »

Then send logs. 

Frédéric RICHARD

**Comment by nunosoares73** [comment]
Hi Frédéric,

I completed the ELM test, was sent by email to ioscanze@icloud.com

CanZE_Debug_20200420_15535856
CanZE_ECU_20200420_15535856
canze_livedata.fic

Best regards,

Nuno

**Comment by FredLeudon** [comment]
Ok, thanks, will have a look soon. 

Regards. 

Frédéric RICHARD

**Comment by nunosoares73** [comment]
Hi Frédéric,

Sorry for the insistence, but you have news from the logs?

Regards,

Nuno

**Comment by FredLeudon** [comment]
Sorry, I didn’t have time to have a look at logs. Will tri before Monday. 

Regards. 

Frédéric RICHARD

**Comment by FredLeudon** [comment]
Hi, 

I had a look at your log files.

Its looks like Your dongle is not fully compatible.

You can have a look at https://canze.fisch.lu/elm327/ <https://canze.fisch.lu/elm327/> for more info.

This one would  work (no guarantee)  https://www.amazon.fr/Lecteur-scanner-Diagnostic-travail-Android/dp/B07B93MYX9 <https://www.amazon.fr/Lecteur-scanner-Diagnostic-travail-Android/dp/B07B93MYX9> 

There are lots of link to find ODB2 dongle in this thread : https://renault-zoe.forumpro.fr/t8555-canze-pour-iphone <https://renault-zoe.forumpro.fr/t8555-canze-pour-iphone>

Regards

Cordialement,


Frédéric RICHARD

**Comment by nunosoares73** [comment]
Hi Frédéric,

Unfortunately, neither of the two links you shared from Amazon (FR) delivers to Portugal.

From your experience and opinion, do you think this will be a good option?

https://pt.banggood.com/KONNWEI-KW902-WiFi-ELM327-OBD2-OBDII-Car-Diagnostic-Scanner-Reader-For-iOS-Android-p-1461010.html?gmcCountry=PT&createTmp=1&utm_source=googleshopping&utm_medium=cpc_bgcs&utm_content=lijing&utm_campaign=ssc-ptg-0724-usw-all&gclid=EAIaIQobChMI75jtm66N6QIVzEPTCh1lZgdTEAQYASABEgKRLvD_BwE&ID=233&cur_warehouse=CN

Regards,

Nuno

**Comment by yoh-there** [comment]
I assume this is resolved. If not, feel free to reopen
